package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "browserslist-lint",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
Revert it too

Reply:
Changed to 0.3.0 version, based on changelog.